direct/tpm2/tpm2.go

```diff
@@ -355,6 +355,33 @@ type CreateLoadedResponse struct {
 // Response implements the Response interface.
 func (*CreateLoadedResponse) Response() tpm.CC { return tpm.CCCreateLoaded }
 
+// GetRandom is the input to TPM2_GetRandom.
+// See definition in Part 3, Commands, section 16.1
+type GetRandom struct {
+	BytesRequested uint16
```

Comment:
nit: add the description "number of octets to return" from the spec as a comment